From 34979267904cec0c53cad1e1361db733f56dcbf8 Mon Sep 17 00:00:00 2001
From: Florian Festi <ffesti@redhat.com>
Date: Wed, 31 May 2017 10:42:03 +0200
Subject: [PATCH 6/6] Pass proper file index when recording generated
 dependencies

Resolves regression introduced with 8f509d6
Fixes #224

(cherry picked from commit dab75e7f26a08cc6798e4df177adb2dd8d40035c)
---
 build/rpmfc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/rpmfc.c b/build/rpmfc.c
index 7fa8227d0..72f29ddf5 100644
--- a/build/rpmfc.c
+++ b/build/rpmfc.c
@@ -511,7 +511,7 @@ static int rpmfcHelper(rpmfc fc, int ix,
     data.exclude = exclude;
 
     for (int i = 0; i < pac; i++) {
-	if (parseRCPOT(NULL, fc->pkg, pav[i], tagN, 0, dsContext, addReqProvFc, &data))
+	if (parseRCPOT(NULL, fc->pkg, pav[i], tagN, ix, dsContext, addReqProvFc, &data))
 	    rc++;
     }
 
-- 
2.13.1

